common/src/main/java/com/wynntils/wc/utils/WynnItemMatchers.java

```diff
@@ -88,4 +92,21 @@ public static boolean isCosmetic(ItemStack stack) {
         }
         return false;
     }
+
+    public static boolean isMythic(ItemStack stack) {
+        return stack.getDisplayName().getString().contains(ChatFormatting.DARK_PURPLE.toString());
```

Comment:
IMO the check for whether its an item and unidentified should be done in this method as well